examples/starter/template/db/migrate.js

```diff
@@ -1,34 +1,18 @@
 const fs = require('fs')
 const path = require('path')
-const parseArgs = require('minimist')
-const shell = require('shelljs')
+const { DATABASE_URL } = require('util.js')
```

Comment:
```suggestion
const { DATABASE_URL } = require('./util.js')
```
Fixing this I was able to run commands with DATABASE_URL